<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="VuMeter" name="widget_3" native="true">
         <zorder>tabWidget</zorder>
        </widget>
       </item>
       <item>
        <widget class="ZynSpinBox" name="widget_2" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Part Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Browser</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Mixer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>KitEditor</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Arp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>Effects</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addSynthBtn">
           <property name="text">
            <string>AddSynth</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>SubSynth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>PadSynth</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="centerWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="bottomWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1332</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_erase_all"/>
    <addaction name="actionOpen_Parameters"/>
    <addaction name="actionSave_All_Parameters"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Scale_Settings"/>
    <addaction name="actionSave_Scale_Settings"/>
    <addaction name="actionShow_Scale_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionCopyright"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInstrument">
    <property name="title">
     <string>Instrument</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Misc</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUi">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionShow_Parameter_Tree"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInstrument"/>
   <addaction name="menuRecord"/>
   <addaction name="menuMisc"/>
   <addaction name="menuUi"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggle_AddSynth"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_Piano"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionToggle_AddSynth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/pixmaps/add.png</normaloff>:/images/pixmaps/add.png</iconset>
   </property>
   <property name="text">
    <string>toggle AddSynth</string>
   </property>
  </action>
  <action name="actionToggle_Piano">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/pixmaps/piano.png</normaloff>:/images/pixmaps/piano.png</iconset>
   </property>
   <property name="text">
    <string>toggle Piano</string>
   </property>
  </action>
  <action name="actionNew_erase_all">
   <property name="text">
    <string>New (erase all)...</string>
   </property>
  </action>
  <action name="actionOpen_Parameters">
   <property name="text">
    <string>Open Parameters</string>
   </property>
  </action>
  <action name="actionSave_All_Parameters">
   <property name="text">
    <string>Save All Parameters</string>
   </property>
  </action>
  <action name="actionLoad_Scale_Settings">
   <property name="text">
    <string>Load Scale Settings...</string>
   </property>
  </action>
  <action name="actionSave_Scale_Settings">
   <property name="text">
    <string>Save Scale Settings...</string>
   </property>
  </action>
  <action name="actionShow_Scale_Settings">
   <property name="text">
    <string>Show Scale Settings...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionCopyright">
   <property name="text">
    <string>Copyright...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionShow_Parameter_Tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Parameter Tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ZynSpinBox</class>
   <extends>QWidget</extends>
   <header>zynspinbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VuMeter</class>
   <extends>QWidget</extends>
   <header>vumeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
